.claude/CLAUDE.md

```diff
@@ -0,0 +1,256 @@
+# ZenML Claude Code Guidelines
+
+This document provides guidance for Claude Code when working with the ZenML codebase. ZenML is an extensible, open-source MLOps framework for creating production-ready ML pipelines.
+
+## Project Structure
+
+- `/src/zenml/` - Core source code
```

Comment:
How about the missing modules here? Some of the important ones that I was able to see on the list were:
- annotators
- alerters
- code repositories
- experiment trackers
- io
- logging
- metadata
- zen server

Reply:
I think we maybe don't want to add *too* many of them, since it's agentic and it should react to the code it finds etc, but you're right maybe some of these need adding like zen server esp.